ckanext/xroad_integration/cli.py

```diff
@@ -74,16 +74,6 @@ def fetch_distinct_service_stats(ctx, days):
         utils.fetch_distinct_service_stats(days)
 
 
-@xroad.command()
-@click.pass_context
-@click.option(u'--days', type=int)
-def fetch_list_errors(ctx, days):
-    'Fetches X-Road errors from catalog lister'
-    flask_app = ctx.meta["flask_app"]
-    with flask_app.test_request_context():
-        utils.fetch_list_errors(days)
-
-
```

Comment:
If you remove CLI commands that fetch the errors from xroad catalog, the errors are not fetched at all.

Reply:
This was for listErrors, the previous error fetching is still done in "def fetch_errors(ctx)"